config/config.go

```diff
@@ -31,7 +32,7 @@ const (
 // TerragruntConfig represents a parsed and expanded configuration
 type TerragruntConfig struct {
 	Description    string              `hcl:"description"`
-	RunConditions  *RunConditions      `hcl:"run_conditions"`
+	RunConditions  RunConditions       `hcl:"run_conditions"`
```

Comment:
Without a pointer, it will create an object with empty lists when it is not in the config, right?

Reply:
Yes.